### DISCUSSION TITLE
Request to deploy ERC20TransferableReceivable on Sepolia

### DISCUSSION THREAD

**Discussion by ariesgun** [RequestNetwork/requestNetwork#1415] [Questions] [answered]
As shown here https://docs.request.network/get-started/supported-chains, the contract has not been deployed yet on Sepolia.
Can I request it to be deployed on this chain?

**Comment by MantisClone** [comment, marked as answer]
Hello @ariesgun 

The ERC20TransferableReceivable was deployed on Sepolia in https://github.com/RequestNetwork/requestNetwork/pull/1418 :white_check_mark: 

Make sure to upgrade to the latest `-next` release of the SDK packages with commands like

```console
npm install @requestnetwork/request-client.js@next
```
